### ISSUE TITLE
Should we rename `ttype` to `type`?

### ISSUE THREAD

**Issue by certik** [rebcabin/masr#9]
To be consistent with just `symbol`, `expr`, etc.

**Comment by certik** [comment]
I named it `ttype` to not clash with some languages that have `type` as a reserved keyword.

**Comment by rebcabin** [comment]
No. "type," like "name," is so heavily overloaded that it loses meaning  quickly. In a big code-base, there may be dozens of things called "type" and dozens of things called "name." Clojure itself uses the name "type" for multiple, subtly different things. That overloading is a source of misdirection, confusion, and loss of time. 

Let's keep it "ttype"

**Comment by certik** [comment]
Very good. Do not close any of these "design issues", I will document this decision and then close it.
